client/src/controllers/DropdownController.ts

```diff
@@ -307,6 +307,13 @@ export class DropdownController extends Controller<HTMLElement> {
   }
 
   disconnect() {
-    this.tippy?.destroy();
+    if (this.tippy) {
+      // Re-add the popper content to the DOM so it can be re-instantiated later if needed.
+      // This is required to support moving elements on the page.
+      this.element?.append(this.tippy.popper);
+      this.tippy.popper.hidden = true;
+      // Then destroy the tippy instance.
+      this.tippy.destroy();
+    }
```

Comment:
This does the same logic as `keepMounted` but as part of disconnect. So this could possibly replace `keepMounted`.

Reply:
This seems to work in most cases, but I managed to replicate the same issue a few times on Firefox. Strangely, when I tried to write down the minimal steps to reproduce the issue, I couldn't trigger it again 🥲

Still, if you do enough drag n drop and interacting with a chooser dropdown, I found that Tippy.js will complain about the tippy not being reachable via keyboard (you can see a large warning in the console). I'm not sure if it's a false positive or not, so we should investigate that before fully removing `keepMountedValue`.

I think I'm happy with this fix for now. I'll see if I can trigger it again during the feature freeze.